pos_sale_order/demo/pos_config_data.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
```

Comment:
This is not demo data but configuration data. Please rename the directory "demo" in "data" to avoid confusion. Thanks